recipes-devtools/python/python3-pyserial_3.4.0.1.bb

```diff
@@ -4,7 +4,7 @@ LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=d476d94926db6e0008a5b3860d1f5c0d"
 
 inherit pypi setuptools3
 
-PYPI_SRC_URI = "https://github.com/labgrid-project/pyserial/archive/v${PV}.tar.gz"
+PYPI_SRC_URI = "https://github.com/labgrid-project/pyserial/releases/download/v3.4.0.1/pyserial_v3.4.0.1.tar.gz"
```

Comment:
Why not using ${PV} here?

Reply:
Pyserial was close to release, it had a 3.5 beta and I didn't check in final release was fine. It might be good to check.

Reply:
Pyserial 3.5.0 does not contain the fix for RFC2217, I have pinged the maintainer on the PR. I will do a rebase and rerelease of the downstream pyserial when 3.5.0 is released without the fix.

Reply:
To get the stable URL I reuploaded the archive as a release, FYI. URL uses `${PV}` now.